oaknational/Oak-Web-Application
# [cur-565] Units table and unit details in generated `.docx`

## Description

- Interleaved units table into unit details in curriculum `.docx`
- Changes to the units, broken down into year/tier/subject
- Also small changes to unit details in curriculum `.docx`

## Issue(s)

`CUR-565` & `CUR-567`

## How to test

1. Go to https://deploy-preview-2448--oak-web-application.netlify.thenational.academy/teachers/curriculum/docx-poc/select
2. Choose a subject via the autoComplete
3. Download the docx template - linked on the page
4. Download the rendered document
5. Check the unit table and unit details are interleaved and contain the correct data

## Screenshots

<img width="502" alt="Screenshot 2024-05-23 at 12 06 26" src="https://github.com/oaknational/Oak-Web-Application/assets/235915/5d0f0b92-d9fc-4fbb-96a4-a27c1015e742">

<img width="507" alt="Screenshot 2024-05-23 at 18 08 39" src="https://github.com/oaknational/Oak-Web-Application/assets/235915/46ba7199-37f4-4c5e-bd2e-93f45d995700">


## Checklist
